src/python/serenity_types/pricing/derivatives/options/volsurface.py

```diff
@@ -80,9 +106,9 @@ class VolatilitySurfaceAvailability(BaseModel):
     Description of the particular volsurface parameters that are available to load.
     """
 
-    build_times: List[datetime]
+    as_of_times: List[datetime]
```

Comment:
I think for this, as_of_time is more meaningful than build_time?

Reply:
OK